index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+
+<html lang="en">
```

Comment:
Kindly I suggest that you remove empty lines in your code base 😸 This will make your code more readable.